extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfConfig.java

```diff
@@ -89,67 +87,59 @@ public class GelfConfig {
      * quarkus.log.handler.gelf.additional-field.field1.type=String
      * </pre>
      */
-    @ConfigItem
     @ConfigDocMapKey("field-name")
-    @ConfigDocSection
-    public Map<String, AdditionalFieldConfig> additionalField;
+    Map<String, AdditionalFieldConfig> additionalField();
```

Comment:
Why did you remove the config doc section?

Reply:
Sorry, mistake